index.html

```diff
@@ -45,6 +45,133 @@ <h2 class="tx-center">Full-Stack Software Developer</h2>
+        <li class="li-documentation">Github</li>
+        <li class="li-documentation">Codepen</li>
+        <li class="li-documentation">Gitlab</li>
+        <li class="li-documentation">Terminal</li>
+      </ul>
+    </div>
+  </section>
+  <!--Documentation section-->
+  <hr>
+  <div class="j-content">
+    <img class="icons" src="img/github-icon.svg" alt="Github logo">
+    <img class="icons" src="img/two-fingers-up.svg" alt="hand oing peace and love">
+    <img class="icons" src="img/twitter-icon.svg" alt="twitter logo">
+    <img class="icons" src="img/medium-icon.svg" alt="medium logo">
+  </div>
+  <div class="blur-circle bc-small"></div>
+  <!--My Work section-->
+  <section id="myWork">
+    <div class="j-content">
+      <ion-icon name="arrow-down-outline"></ion-icon>
+      <h5>WORK</h5>
+    </div>
+    <h3>MY RECENT<br>WORK</h3>
     <hr>
-
-</body>
\ No newline at end of file
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
+      <div class="blur-elipse-2"></div>
+    </div>
+    <div class="g-container">
+      <img class="g-item1" src="img/Snapshoot-Portfolio.png" alt="Imagen de prueba">
+      <h4 class="g-item2">Multi-Post Stories</h4>
+      <p class="g-item3">
+        A daily selection of privately personalized reads; 
+        no accounts or sign-ups required. has been the industrys standard 
+        dummy text ever since the 1500s, when an unknown printer took a 
+        standard dummy text.
+      </p>
+      <a class="g-item4">css</a>
+      <a class="g-item5">html</a>
+      <a class="g-item6">bootstrap</a>
+      <a class="g-item7">Ruby</a>
+      <button class="g-item8" type="button">See Project</button>
```

Comment:
- [ ] The buttons should be a replica of the original design. Kindly adjust the button to match the design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153714356-b7f0b6df-41df-4f38-b019-c4b905ca0136.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153714370-8a436c1f-9906-45c4-99bf-a26f60319b32.png)

Reply:
About my design

![image](https://user-images.githubusercontent.com/98621809/153731605-5da51fb5-2ba5-42d5-9247-341609693417.png)

Figma design

![image](https://user-images.githubusercontent.com/98621809/153731615-d2fb9fe9-0069-4f35-aeec-da3e32b162a6.png)

Reply:
- Nice work, implementing the required changes 💪🏿 

![375](https://user-images.githubusercontent.com/69638013/153733400-d5e0f93c-86e4-46d2-a517-3d3a94e48f2a.PNG)
-  On 375 the last li goes to the second line

- [ ] Kindly adjust the padding so that your design can accomodate other small screen sizes 👍🏿